custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/score.R

```diff
@@ -55,14 +56,18 @@ has_read_input_data_hook <- function() {
 #' @export
 #'
 #' @examples
-load_serialized_model <- function(model_dir) {
+load_serialized_model <- function(model_dir, target_type) {
     model <- NULL
     if (!isFALSE(load_model_hook)) {
         model <- load_model_hook(model_dir)
     }
     if (is.null(model)) {
         file_names <- dir(model_dir, pattern = CUSTOM_MODEL_FILE_EXTENSION)
         if (length(file_names) == 0) {
+            # Allow no serialized models if it is a transform
+            if (target_type == TargetType$TRANSFORM) {
```

Comment:
do we allow users to not have a fit method when they are doing transforms? If we don't require serialized models, maybe we should do that as well.

Reply:
We currently always require fit but yes we should make it optional for transforms. I'll make a ticket for that in a follow-up because it will require some extra effort to pass target type into drum fit

Reply:
https://datarobot.atlassian.net/browse/RAPTOR-5847

Reply:
I think it's the case for python too